ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -117,8 +113,8 @@ class ColorDemoController: UIViewController {
         view.addSubview(stackView)
         view.backgroundColor = Colors.NavigationBar.background
 
-        let window = ColorDemoController.themeWindowType
-        segmentedControl.selectedSegmentIndex = colorProviderThemes.firstIndex(where: { $0.demoColorTheme.windowType == window }) ?? 0
+        let demoListViewController = navigationController?.viewControllers.first as? DemoListViewController
+        segmentedControl.selectedSegmentIndex = DemoColorTheme.allCases.firstIndex(where: { $0.name == demoListViewController!.themeName }) ?? 0
```

Comment:
Here's another chance to get rid of a force unwrap (`!`). If you use an `if let` when assigning `demoListViewController` then you'll be able to use it without unwrapping, and just handle the default case in an `else` block afterwards.